File: src/main/java/client/ClientWorld.java
package client;

import common.world.World;

import java.awt.*;

public class ClientWorld extends World {
    public ClientWorld(String name, int x, int y) {
        super(name, x, y);
    }

    /** Worlds shouldn't be doing any drawing. your Render manager should be **/
    @Deprecated
    public void draw(Graphics g) {

    }

    @Override
    public void update() {

    }
}

File: src/main/kotlin/client/ClientWorld.kt
package client

import common.world.World

import java.awt.*

class ClientWorld(name: String, x: Int, y: Int) : World(name, x, y) {

    /** Worlds shouldn't be doing any drawing. your Render manager should be  */
    @Deprecated("")
    fun draw(g: Graphics) {}

    override fun update() {}
}
